test/components/structures/RoomView-test.tsx

```diff
@@ -93,16 +94,20 @@ describe("RoomView", () => {
             await switchedRoom;
         }
 
+        const stores = new Stores(RoomViewStore.instance);
+        stores.client = cli;
```

Comment:
Maybe we can make the client passable as an optional last arg to clean up test cases